docs/extend/extend-desktop/mvd-buildingplugins.md

```diff
@@ -1,17 +1,19 @@
-# Building plugins
-You can build a plugin by using the following steps as a model. Static web content must be in a /web directory, and server and other backend files must be in a /lib directory. You can place other plugin source code anywhere.
+# Building plugin apps
+You can build a plugin app by using the following steps as a model. Alternatively, you can follow the [Sample Angular App tutorial](https://github.com/zowe/sample-angular-app/blob/lab/step-1-hello-world/README.md).
```

Comment:
About "plugin app". Some of our documentation says "plug-in" versus "plugin". But also, some plugins aren't "apps" - such as security plugins. So, to be technically correct the guide below is specifically for Application-type plugins.